app/components/login_button_component.scss

```
@@ -0,0 +1,52 @@
+@use 'uswds-core' as *;
+
+.login-button.usa-button--big > .login-button__logo {
+  font-size: 1.8rem;
+  margin-top: -2px;
+}
+
+.login-button__logo {
+  margin-left: 2px;
+  font-size: 1.45rem;
+  vertical-align: middle;
+  color: transparent;
+  background: no-repeat 100% url('logo.svg');
+}
+
+.login-button__logo-white {
+  background-image: url('logo-white.svg');
+}
```

I think we could avoid the additional class `.login-button__logo-white` (which doesn't feel like valid BEM) by using the modifier class where we expect the styles to apply, i.e.


```suggestion

  .login-button--primary-darker & {
    background-image: url('logo-white.svg');
  }
}
```